website/cue/reference/components/sinks/base/splunk_hec_logs.cue

```diff
@@ -5,6 +5,18 @@ base: components: sinks: splunk_hec_logs: configuration: {
 		description: "Splunk HEC acknowledgement configuration."
 		required:    false
 		type: object: options: {
+			cookie_name: {
+				description: """
+					The name of a cookie to extract from the Splunk HEC response to use when querying for acknowledgements.
```

Comment:
```suggestion
					Specifies the name of a cookie to extract from the Splunk HEC response and use when querying for acknowledgements.
```

Reply:
@estherk15  Thanks for the review! I've changed this and added a bit more text describing why a cookie is useful here (specifically that it is useful for routing the request to the correct indexer)